common/src/View/QtUtils.cpp

```diff
@@ -73,6 +71,31 @@ namespace TrenchBroom {
             m_widget->setUpdatesEnabled(true);
         }
 
+        /**
+         * Helper function to find an existing settings file, or build the path to such a file if it doesn't exist yet.
+         */
+        QString getSettingsFilePath();
+        QString getSettingsFilePath() {
+            auto path = QStandardPaths::locate(QStandardPaths::ConfigLocation, QString::fromLocal8Bit("TrenchBroom Preferences"));
+            if (path.isEmpty()) {
+                // if the file does not exist, it cannot be located
+                path = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/Trenchbroom Preferences";
+            }
+            return path;
+        }
+
+        QSettings& getSettings() {
+            static auto settings =
```

Comment:
Return the QSettings by value, and remove the static. The docs specifically say it's very fast to create QSettings, and all Qt example code creates them whenever reading/writing a setting.  https://doc.qt.io/qt-5/qsettings.html#basic-usage

Reply:
But isn't it better to remove this method altogether and just use `QSettings` directly everywhere?

Reply:
Alright, your comment above indicates that you think so, too. I'll remove this method altogether and use `QSettings()` everywhere.